automation/check-patch.e2e-k8s.sh

```diff
@@ -15,7 +15,6 @@ main() {
     export KUBEVIRT_PROVIDER='k8s-1.14.6'
 
     source automation/check-patch.e2e.setup.sh
-    cd ${TMP_PROJECT_PATH}
```

Comment:
Why do you remove this? It is a part of the kubevirtci scripting. It makes sure that you can execute the automation/ test script on any environment and it will create a temporary project directory with everything working, even if you did not have a working GOPATH before.

Reply:
Adding them it back.